frontend/src/metabase/visualizations/components/ChartClickActions.jsx

```diff
@@ -148,7 +148,7 @@ class ChartClickActions extends Component {
     const groupedClickActions = _.groupBy(clickActions, "section");
     if (groupedClickActions["sum"] && groupedClickActions["sum"].length === 1) {
       // if there's only one "sum" click action, merge it into "summarize" and change its button type and icon
-      groupedClickActions["summarize"].push({
+      groupedClickActions?.["summarize"]?.push({
```

Comment:
I'm not quite sure what this does, but it throws errors and crashes the application if you don't have any drill options without the null chaining operators.

Reply:
What's the step to reproduce the crash?

Reply:
On master, if you add `return [];` to the top of the `ColumnFilterDrill` function (to force it empty)

![Screen Shot 2022-07-18 at 8 36 13 AM](https://user-images.githubusercontent.com/30528226/179535663-bb9da801-b5a9-4d3f-acba-2a1bad1a148b.png)

And you open a json column (and primary key columns I think), you get this error:

![Screen Shot 2022-07-18 at 8 35 27 AM](https://user-images.githubusercontent.com/30528226/179535771-7270f9a4-0a7e-4991-a9dd-92f210cdd51b.png)

Because it is trying to force the `Distinct Values` action into the `summarize` section that does not exist.

I pushed a bit better change, so instead of null-chain-ignoring the push, if the summarize section is empty, we create it.

![image](https://user-images.githubusercontent.com/30528226/179540143-b7569a3a-5334-4c78-953d-352df7c089f9.png)